### ISSUE TITLE
Error running bundle exec danger init: No recent pull requests found for this repo

### ISSUE THREAD

**Issue by kennethlynne** [danger/danger#274]
I have run through all steps successfully except the final one (exposing the GitHub token as an environment variable).

It fails with a mysterious error message: 
![image](https://cloud.githubusercontent.com/assets/390817/16010362/2dc6acec-3181-11e6-89cb-51a534d5cfa3.png)

I have several ongoing pull requests on the repository in mention, and I am not sure how to move forward on this.

**Comment by justMaku** [comment]
When running with local git repo you need to specify the *LOCAL_GIT_PR_ID" environment variable. It's strange though that it tries to run into local mode. Will look into that.

**Comment by kennethlynne** [comment]
@justMaku 
![image](https://cloud.githubusercontent.com/assets/390817/16015892/1ebe2830-3198-11e6-849d-cb089740312d.png)

**Comment by kennethlynne** [comment]
git log --merges --oneline
![image](https://cloud.githubusercontent.com/assets/390817/16015914/327dc272-3198-11e6-8180-e3e78bcf6f8d.png)

**Comment by kennethlynne** [comment]
@justMaku `export LOCAL_GIT_PR=20 && bundle exec danger init` yields
```ruby 
No recent pull requests found for this repo, danger requires at least one PR for the local mode. (RuntimeError)
        /usr/local/lib/ruby/gems/2.3.0/gems/danger-0.8.2/lib/danger/ci_source/local_git_repo.rb:52:in `initialize'
        /usr/local/lib/ruby/gems/2.3.0/gems/danger-0.8.2/lib/danger/commands/init.rb:137:in `new'
        /usr/local/lib/ruby/gems/2.3.0/gems/danger-0.8.2/lib/danger/commands/init.rb:137:in `current_repo_slug'
        /usr/local/lib/ruby/gems/2.3.0/gems/danger-0.8.2/lib/danger/commands/init.rb:235:in `circle_token'
        /usr/local/lib/ruby/gems/2.3.0/gems/danger-0.8.2/lib/danger/commands/init.rb:160:in `setup_danger_ci'
        /usr/local/lib/ruby/gems/2.3.0/gems/danger-0.8.2/lib/danger/commands/init.rb:35:in `run'
```

**Comment by justMaku** [comment]
Sorry, the environment variable name is actually `LOCAL_GIT_PR_ID`.

**Comment by kennethlynne** [comment]
@justMaku `export LOCAL_GIT_PR_ID=20 && bundle exec danger init`
yields the exact same error :C

**Comment by justMaku** [comment]
@kennethlynne right, that wouldn't work with your setup, please try using `fix_init_problem_with_repos_without_merges` branch for now.

**Comment by justMaku** [comment]
Fixed in #275
